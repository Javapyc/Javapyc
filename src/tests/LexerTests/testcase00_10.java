/* check for combinations of reserved words and operators
*/
class*else<static>=extends&&voidint{=,main][while.String][||!==return/true;false-this!new(if)false=boolean+int*null||this!=true=false!new,staticnew<static>==!=
