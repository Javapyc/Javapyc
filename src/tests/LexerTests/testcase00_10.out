ReservedWord, class
Operator, *
ReservedWord, else
Operator, <
ReservedWord, static
Operator, >=
ReservedWord, extends
Operator, &&
ID, voidint
Delimiter, {
Delimiter, =
Delimiter, ,
ReservedWord, main
Delimiter, ]
Delimiter, [
ReservedWord, while
Delimiter, .
ReservedWord, String
Delimiter, ]
Delimiter, [
Operator, ||
Operator, !=
Delimiter, =
ReservedWord, return
Operator, /
ReservedWord, true
Delimiter, ;
ReservedWord, false
Operator, -
ReservedWord, this
Operator, !
ReservedWord, new
Delimiter, (
ReservedWord, if
Delimiter, )
ReservedWord, false
Delimiter, =
ReservedWord, boolean
Operator, +
ReservedWord, int
Operator, *
ReservedWord, null
Operator, ||
ReservedWord, this
Operator, !=
ReservedWord, true
Delimiter, =
ReservedWord, false
Operator, !
ReservedWord, new
Delimiter, ,
ID, staticnew
Operator, <
ReservedWord, static
Operator, >=
Delimiter, =
Operator, !=
